src/screens/ChatScreen/components/ChatHeader.js

```diff
@@ -46,6 +55,14 @@ const MenuIcon = style => {
   return <Icon {...style} name="more-vertical" height={40} width={20} />;
 };
 
+const ResolveIcon = style => {
+  return <Icon {...style} fill="#13ce66" name="checkmark-circle-outline" height={40} width={20} />;
```

Comment:
Add this color in theme configuration https://github.com/chatwoot/chatwoot-mobile-app/blob/develop/src/theme.js